mp/src/game/client/viewpostprocess.cpp

```diff
@@ -2963,6 +2963,9 @@ void DoEnginePostProcessing( int x, int y, int w, int h, bool bFlashlightIsOn, b
 				DoThermalVision(x, y, w, h);
 				break;
 			}
+			case NEO_CLASS_VIP:
+				// Cry a little
```

Comment:
nit: do we intend on giving VIP a vision mode?

Reply:
start a discussion about it I guess